layouts/partials/fragments/contact.html

```diff
@@ -142,8 +142,9 @@ <h2>{{- . | markdownify -}}</h2>
                     </button>
                     <span class="filler"></span>
                   </div>
+                  <div data-netlify-recaptcha></div>
```

Comment:
I think we either should display netlify recaptcha or normal recaptcha. Displaying both is confusing.

Reply:
Okay. Yeah, so that's why I needed a clearer description of the task. @stp-ip  was talking about the options available and the one we should go with. any ideas @stp-ip?

Reply:
Maybe having custom recaptcha + netlify as option is too complicated and we should just go with:
* netlify, netlify + netlify recaptcha
* custom, custom + custom recaptcha

Reply:
I think this is a better approach. Should we close this PR and move to #257 ?